index.html

```diff
@@ -9,6 +9,23 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="book-container column">
+    <div class="row">
+        <h1 class="book-title">Título del libro</h1>
+        <p class="book-gender">(género)</p>
+    </div>
+    <div class="row">
+        <h3 class="book-author text-1">Autor del libro:</h3>
+        <p class="book-description text-1-description">Nombre del autor del libro</p>
```

Comment:
Why don't we use `text-2` since the idea is to be semantic about this. Different types of texts. With that in mind, adding `description` makes it lose its meaning